app/front_assets/components/SearchInternshipOffer.jsx

```diff
@@ -3,17 +3,20 @@ import Turbolinks from 'turbolinks';
 
 import CityInput from './inputs/CityInput';
 import KeywordInput from './inputs/KeywordInput';
+import SchoolTypeInput from './inputs/SchoolTypeInput';
 
 import findBootstrapEnvironment from '../utils/responsive';
 
-function SearchInternshipOffer({ url, initialLocation, className }) {
+function SearchInternshipOffer({ url, schoolTypeVisibility=true, className }) {
```

Comment:
```suggestion
function SearchInternshipOffer({ url, className, schoolTypeVisibility=true }) {
```
les params ac un default a la fin